crates/symbolicator-service/tests/integration/snapshots/integration__symbolication__wasm_payload.snap

```diff
@@ -49,4 +49,8 @@ modules:
         location: "http://localhost:<port>/symbols/bd/a18fd85d4a4eb893022d6bfad846b1.src.zip"
         download:
           status: notfound
+      - source: local
+        location: "http://localhost:<port>/symbols/demo.wasm/BDA18FD85D4A4EB893022D6BFAD846B10/demo.wasm"
```

Comment:
I’m wondering why a completely unrelated filetype now has more candidates? 🤔

Reply:
yeah that was weird but my assumption is that it's related to symbolic update

Reply:
possible. trying yet another file name pattern does definitely not hurt, though I would still love to find out why this suddenly happens.